src/test/java/com/ericsson/ei/handlers/test/RmqHandlerTest.java

```diff
@@ -123,4 +129,13 @@ public void testMessageBusConnection() {
         factory = rmqHandler.connectionFactory();
         assertNotNull(factory);
     }
+
+    @Test
+    public void testIsRabbitMQServerUp() {
+        Mockito.when(container.getActiveConsumerCount()).thenReturn(1);
```

Comment:
Here I would also prefer to not mock anything outside RMQHandler.

Reply:
Same as before, then it should be a functional test

Reply:
Yes. You can leave the unit test cases you implemented as they are and add functional tests.